backend/pkg/kubeconfig/kubeconfig.go

```diff
@@ -910,20 +910,58 @@ func GetInClusterContext(oidcIssuerURL string,
 	}, nil
 }
 
+// Func type for context filter, if the func return true,
+// the context will be skipped otherwise the context will be used.
+type shouldBeSkippedFunc = func(kubeContext Context) bool
+
+// AllowAllKubeContext will keep all contexts.
+func AllowAllKubeContext(_ Context) bool {
+	return false
+}
+
+// SkipKubeContextInCommaSeperatedString will skip the contexts
+// whose names are in the given comma-separated string.
+// For example, if pass the "a,b" for blackKubeContextNameStr,
+// the contexts named "a" and "b" will be skipped.
+func SkipKubeContextInCommaSeperatedString(blackKubeContextNameStr string) shouldBeSkippedFunc {
```

Comment:
There's a typo in SkipKubeContextInCommaSeperatedString. It should be Separated not Seperated.

Reply:
thanks for your carefulness, updated ~